web/src/main/java/com/epam/esm/configuration/JacksonConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.esm.configuration;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.openapitools.jackson.nullable.JsonNullableModule;
+import org.springframework.context.annotation.Configuration;
+
+import javax.annotation.PostConstruct;
+
+@Configuration
+public class JacksonConfiguration {
+
+    private final ObjectMapper objectMapper;
+
+    public JacksonConfiguration(ObjectMapper objectMapper) {
+        this.objectMapper = objectMapper;
+    }
+
+    @PostConstruct
+    ObjectMapper jacksonObjectMapper() {
+        objectMapper.registerModule(new JsonNullableModule());
```

Comment:
have you already checked this already https://docs.spring.io/spring-boot/docs/current/reference/html/howto.html#howto-customize-the-jackson-objectmapper ?